src/turbopuffer.ts

```diff
@@ -219,14 +252,34 @@ export class Namespace {
     vectors: Vector[];
```

Comment:
can we delete and upsert in the same call with this type? the api technically supports it, but I'm not sure the ts types allow it

Reply:
Went ahead and updated the relevant TS type to allow this, probably a reasonable use case for advanced users